common/lib/xmodule/xmodule/js/src/video/03_video_player.js

```diff
@@ -553,6 +553,11 @@ function (HTML5Video, Resizer) {
         // `duration`. In this case, slider doesn't reach the end point of
         // timeline.
         this.videoPlayer.updatePlayTime(time);
+
+        // Emit 'pause_video' event when a video ends if Player is of Youtube
+        if (this.isYoutubeType()) {
```

Comment:
Where in the code does the HTML5 video emit the event? I guess I'm wondering why there are two different code paths for HTML5 and youtube...